test/DotNetty.Transport.Libuv.Tests/ConnectionAttemptTests.cs

```diff
@@ -44,7 +44,7 @@ static void ConnectTimeout0(Bootstrap cb)
             IPAddress ipAddress = IPAddress.Parse("198.51.100.254");
             var badAddress = new IPEndPoint(ipAddress, 65535);
             Task<IChannel> task = cb.ConnectAsync(badAddress);
-            var error = Assert.Throws<AggregateException>(() => task.Wait(DefaultTimeout));
+            var error = await Assert.ThrowsAsync<AggregateException>(async () => task.Wait(DefaultTimeout));
```

Comment:
This is just a hack because xUnit does this internally:
```
protected static Exception RecordException(Func<object> testCode)
{
  Assert.GuardArgumentNotNull(nameof (testCode), (object) testCode);
  Task task;
  try
  {
    task = testCode() as Task;
  }
  catch (Exception ex)
  {
    return ex;
  }
  if (task != null)
    throw new InvalidOperationException("You must call Assert.ThrowsAsync, Assert.DoesNotThrowAsync, or Record.ExceptionAsync when testing async code.");
  return (Exception) null;
}
```

Once it passes I'll see if there is a better way